clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/KafkaConsumerService.java

```diff
@@ -341,6 +350,7 @@ public long getMaxElapsedTimeMSSinceLastPollInConsumerPool() {
         slowestTaskId = task.getTaskId();
       }
     }
+    aggStats.recordTotalConsumerIdleTime(maxElapsedTimeSinceLastPollInConsumerPool);
```

Comment:
This is not what you said in the Slack discussion about "deduce this is to look at the gaps of data points for store-level byte_per_poll metrics", is that right? Could you please add some more explanation here?

Reply:
This is for `total` stats for the given region, so this metric does tell us the max idle time among all consumers in a given region.

Reply:
@sixpluszero  can you please review again? I've updated the logic to scan all partitions in SIT as you suggested.